webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpaceCardFront.vue

```diff
@@ -367,6 +367,16 @@ export default {
         }, this.waitTimeUntilCloseMenu);
       }
     });
+    document.addEventListener('favorite-added', event => {
+      if (event && event.detail) {
+        this.space.isFavorite = 'true';
+      }
+    });
```

Comment:
```suggestion
    document.addEventListener('space-favorite-added', event => {
      if (event && event.detail === this.space.id) {
        this.space.isFavorite = 'true';
      }
    });
```

Reply:
Done for the verification space, but for the event name, it is already created and used with this title.
Do I have to change it for every use of the event?

Reply:
Done!